cockatrice/src/dlg_forgotpassword.cpp

```diff
@@ -0,0 +1,67 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpassword.h"
+
+DlgForgotPassword::DlgForgotPassword(QWidget *parent)
+    : QDialog(parent)
+{
```

Comment:
My question for this dialog is can we possibly re-use the server login portion with the dropdown menu so people don't always have to type things in constantly?